lib/post.js

```diff
@@ -40,16 +40,16 @@ function postXPI(manifest, options) {
           reject(e);
         });
         client.on("close", function(hadError) {
-          console.log(hadError ? "Posting XPI to %s failed" :
+          console.log(hadError ? "Posting XPI to %s failed" : // eslint-disable-line no-console
```

Comment:
These should be switched to the logger. You can set it up like this at the top of the module:
````
var utils = require("./utils");
var logger = utils.console;
````
And use it like either `logger.info(...)` or `logger.error(...)`